packages/esm-patient-medications-app/src/order-basket/order-basket.component.tsx

```diff
@@ -131,64 +135,60 @@ const OrderBasket = connect<OrderBasketProps, OrderBasketStoreActions, OrderBask
     );
   };
 
+  if (isLoading) return <InlineLoading className={styles.loader} description={t('loading', 'Loading...')} />;
+
+  if (isMedicationOrderFormVisible) {
+    return (
+      <MedicationOrderForm
+        initialOrderBasketItem={medicationOrderFormItem}
+        onSign={onMedicationOrderFormSigned}
+        onCancel={() => setIsMedicationOrderFormVisible(false)}
+      />
+    );
+  }
+
   return (
     <>
-      {(() => {
-        if (isLoading) return <InlineLoading className={styles.loader} description={t('loading', 'Loading...')} />;
-        if (isMedicationOrderFormVisible) {
-          return (
-            <MedicationOrderForm
-              initialOrderBasketItem={medicationOrderFormItem}
-              onSign={onMedicationOrderFormSigned}
-              onCancel={() => setIsMedicationOrderFormVisible(false)}
-            />
-          );
-        }
-        return (
-          <>
-            <OrderBasketSearch encounterUuid={encounterUuid} onSearchResultClicked={handleSearchResultClicked} />
-            <div className={styles.container}>
-              <div className={styles.orderBasketContainer}>
-                <OrderBasketItemList
-                  orderBasketItems={items}
-                  onItemClicked={(order) => openMedicationOrderFormForUpdatingExistingOrder(items.indexOf(order))}
-                  onItemRemoveClicked={(order) => {
-                    const newOrders = [...items];
-                    newOrders.splice(items.indexOf(order), 1);
-                    setItems(newOrders);
-                  }}
+      <OrderBasketSearch encounterUuid={encounterUuid} onSearchResultClicked={handleSearchResultClicked} />
+      <div className={styles.container}>
+        <div className={styles.orderBasketContainer}>
+          <OrderBasketItemList
+            orderBasketItems={items}
+            onItemClicked={(order) => openMedicationOrderFormForUpdatingExistingOrder(items.indexOf(order))}
+            onItemRemoveClicked={(order) => {
+              const newOrders = [...items];
+              newOrders.splice(items.indexOf(order), 1);
+              setItems(newOrders);
+            }}
+          />
+          {(() => {
+            if (isLoadingOrders) return <DataTableSkeleton role="progressbar" />;
+            if (isError) return <ErrorState error={isError} headerTitle={headerTitle} />;
```

Comment:
`ErrorState` prefers if the actual error response object is passed, but this error is named like it should be a boolean. Would it be appropriate here to pass the error response object instead? Or label the variable so we don't think its going to be a boolean?

Reply:
Yes, that variable is wrongly named. I've amended that in the latest commit.